.gitignore

```diff
@@ -15,3 +15,5 @@ bin/
 
 /text-ui-test/ACTUAL.TXT
 text-ui-test/EXPECTED-UNIX.TXT
+
```

Comment:
new line is weird but I'll allow it

Reply:
to clarify further this is ok (and sometimes even encouraged), it just caught me off guard